branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -51,4 +51,58 @@ public IBranchLayout slideOut(String branchName) throws BranchLayoutException {
       return List.of(entry.withChildren(children.flatMap(child -> slideOut(child, entryToSlideOut))));
     }
   }
+
+  @Override
+  public IBranchLayout slideIn(String parentBranchName, IBranchLayoutEntry entryToSlideIn) throws BranchLayoutException {
```

Comment:
Ofc doable but what for?

The first thing is that passing `IBranchLayoutEntry` makes sense. 
This is because we slide in the whole entry (with its children entries).

Note that slide in is not "similar" to slide out (children's behavior :baby: :baby: ).

The second thing that passing `parentBranchName` is better than parent entry. 
This is because we want our entry to come from `this` branch layout (therefore we `findEntryByName`).
Passing an entry would increase complexity (shall we compare entry by reference, by name, by children?). 

I think that the present solution is justified.